common/scala/src/main/scala/whisk/connector/kafka/KafkaConsumerConnector.scala

```diff
@@ -48,39 +50,27 @@ class KafkaConsumerConnector(
      * duration.
      *
      * @param duration the maximum duration for the long poll
-     * @param syncCommit update offsets on the topic immediately.  Default false as we use auto-commit.
      */
-    def getMessages(duration: Duration = 500 milliseconds, syncCommit: Boolean = false) = {
+    override def peek(duration: Duration = 500 milliseconds) = {
```

Comment:
renamed to peek - to emphasize need for commit() (even with kafka auto commit, want to commit offsets when we have processed messages eagerly).